### DISCUSSION TITLE
URL trailing slash issue.

### DISCUSSION THREAD

**Discussion by richwalm** [firefly-iii/firefly-iii#4605] [General]
Hi there,

Within the [public/.htaccess](https://github.com/firefly-iii/firefly-iii/blob/main/public/.htaccess) file, there's a rule to strip off trailing slashes if not pointing to a folder.

This was causing an issue on my install. To re-enable the introduction guidance tooltips for the dashboard, it makes an XHR request to 'https://firefly.example.org/json/intro/enable/index/'. Due to the trailing slash on this link and this rewrite rule, Apache returns a 301 redirect rather than a 200 code, causing the script to fail.

Commenting out this redirect rule fixed it for me.

I'm just putting this out there if anyone else has this issue. Seems to be in effect on the Docker image, although not on the demo site.

**Comment by JC5** [comment]
I see it too. Thanks for the deep dive!
